internal/pkg/agent/application/upgrade/marker_watcher_test.go

```diff
@@ -84,6 +84,9 @@ func TestProcessMarker(t *testing.T) {
 	cases := map[string]struct {
 		markerFileContents string
 
+		watcherAgentVersion string
+		watcherAgentHash    string
```

Comment:
These are not actually the watcher agent version and hash (as we discussed off-PR yesterday).  Could you change these to `agentVersion`/`agentHash` or `currentAgentVersion`/`currentAgentHash` or something along those lines please?

Reply:
Fixed in 78cf065a1d6c069f332b3d48734f067076347c89